src/app/components/surveillance/complaints/complaint-view.jsx

```diff
@@ -50,127 +74,144 @@ function ChplComplaintView(props) {
-          <Typography>
-            Complainant Type:
-          </Typography>
-          <Typography>
-            {complaint.complainantType?.name}
-          </Typography>
-          <Typography>
-            Complainant Type (Other):
-          </Typography>
-          <Typography>
-            {complaint.complainantTypeOther}
-          </Typography>
-          <Typography>
-            Complaint Summary:
-          </Typography>
-          <Typography>
-            {complaint.summary}
-          </Typography>
-          <Typography>
-            Associated Criteria:
-          </Typography>
-          {complaint.criteria?.length > 0
-            ? (
-              <ul>
-                {complaint.criteria.map((criterion) => <li key={criterion.id}>{`${criterion.removed ? 'Removed |' : ''} ${criterion.number}: ${criterion.title}`}</li>)}
-              </ul>
-           )
-            : (
-              <Typography>
-                None
-              </Typography>
-           )}
-          <Typography>
-            Actions/Responses:
-          </Typography>
-          <Typography>
-            {complaint.actions}
-          </Typography>
-          <Typography>
-            Associated Certified Products:
-          </Typography>
-          {complaint.listings?.length > 0
-            ? (
-              <ul>
-                {complaint.listings.map((listing) => <li key={listing.id}>{ listing.chplProductNumber }</li>)}
-              </ul>
-           )
-            : (
-              <Typography>
-                None
-              </Typography>
-           )}
-          <Typography>
-            Associated Surveillance Activities:
-          </Typography>
-          {complaint.surveillances?.length > 0
-            ? (
-              <ul>
-                {complaint.surveillances.map((surveillance) => <li key={surveillance.id}>{`${surveillance.chplProductNumber}: ${surveillance.friendlyId}`}</li>)}
-              </ul>
-           )
-            : (
-              <Typography>
-                None
-              </Typography>
-           )}
-          <Typography>
-            Complainant Contacted:
-          </Typography>
-          <Typography>
-            {complaint.complainantContacted ? 'Yes' : 'No'}
-          </Typography>
-          <Typography>
-            Developer Contacted:
-          </Typography>
-          <Typography>
-            {complaint.developerContacted ? 'Yes' : 'No'}
-          </Typography>
-          <Typography>
-            ONC-ATL Contacted:
-          </Typography>
-          <Typography>
-            {complaint.oncAtlContacted ? 'Yes' : 'No'}
-          </Typography>
-          <Typography>
-            Informed ONC per &sect;170.523(s):
-          </Typography>
-          <Typography>
-            {complaint.flagForOncReview ? 'Yes' : 'No'}
-          </Typography>
+        <CardHeader
+        title="Viewing Compliant"
+        subheader={complaint.acbComplaintId}
+        />
+
```

Comment:
Or this space. Generally, in a block of JSX, we don't separate things with blank lines. We do between functions, or "sections" of import statements, but not in JSX or in JS objects